developer-portal/for-developers/bigcommerce/adding-fast-button-to-blog-posts/adding-code.md

```diff
@@ -25,6 +25,42 @@ To add Fast Checkout to your blog posts, you will need to:
 
 ## Adding Code to Your Blog Post
 
+### Option 1: Use Fast Headless Checkout
+
+1. In your newly created blog post, add the following code snippet.
+
+   > You will need to assign a unique button ID (`button_id="{INSERT-UNIQUE-ID}"`) to the button (`fast-checkout-button`), enclosed within a form field, along with (at minimum) an App ID (`app_id`) and a Product ID (`product_id`).
+
+   ```html
+   <form>
+     <fast-checkout-button
+       button_id="UNIQUE_BUTTON_ID"
+       app_id="YOUR_APP_ID"
+       product_id="YOUR_PRODUCT_ID"
+     ></fast-checkout-button>
+   </form>
```

Comment:
`<fast-checkout-button>` no longer needs to be in a `<form>` if the product attributes are provided
```suggestion
   <fast-checkout-button
     button_id="UNIQUE_BUTTON_ID"
     app_id="YOUR_APP_ID"
     product_id="YOUR_PRODUCT_ID"
   ></fast-checkout-button>
```